build.gradle

```diff
@@ -189,29 +194,14 @@ if(skipPrebuildLibraries!="true"&&buildNativeProjects!="true"){
                     String p1=srcRel.substring(0,j);
                     String p2=srcRel.substring(j);
                     if(!p1.equals("android")&&!p2.startsWith("/native"))  srcRel=p1+"/native"+p2;
-                    //
-
                     File dest=new File(nativesPath+File.separator+srcRel);
-
-                    boolean include=false;
-                    if(!dest.exists()){
-                        include=true;
-                        println("Copy "+src+" "+dest);
-                    }else if(dest.lastModified()<src.lastModified()){
-                        include=true;
-                        println("Copy "+src+" "+dest+  ". Source is newer. src "+src.lastModified()+ " dest "+dest.lastModified());
-                    }
-                    else{
-                        println(""+dest+" Up to date. Skip.");
-                    }
-
-                    if(include){
-                        dest.getParentFile().mkdirs();
-                        Files.copy(src.toPath(), dest.toPath(), StandardCopyOption.REPLACE_EXISTING);
-                    }
+                    println("Copy "+src+" "+dest);
+                    dest.getParentFile().mkdirs();
+                    Files.copy(src.toPath(), dest.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
This part changes the behaviour, since it will always replace the files even if the local files are newer. We want this to work also for people that are compiling natives locally.

Reply:
Yes, you are right! I changed the code to respect this requirement.